test-suite-groovy/src/test/groovy/io/micronaut/docs/server/urisyntax/MalformedUriSpec.groovy

```diff
@@ -0,0 +1,27 @@
+package io.micronaut.docs.server.urisyntax
+
+import io.micronaut.context.ApplicationContext
+import io.micronaut.context.env.Environment
+import io.micronaut.http.HttpStatus
+import io.micronaut.runtime.server.EmbeddedServer
+import spock.lang.AutoCleanup
+import spock.lang.Shared
+import spock.lang.Specification
+
+class MalformedUriSpec extends Specification {
+
+    @Shared
+    @AutoCleanup
+    EmbeddedServer embeddedServer = ApplicationContext.run(EmbeddedServer, ['spec.name': MalformedUriSpec.simpleName], Environment.TEST);
```

Comment:
Since you are not creating beans in this test. You don't need to set `spec.name`. We typically use `spec.name` to avoid bean pollution. 

Environment.TEST is automatically inferred. 

Thus, you could just write `EmbeddedServer embeddedServer = ApplicationContext.run(EmbeddedServer)`

Reply:
Fixed! thanks for the point.